src/main/java/com/exadel/sober/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.exadel.sober.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler({NoSuchUserExistsException.class,
+            UserAlreadyExistsException.class,
+            BadUserCredentialException.class,
+            CannotAddReasonException.class,
+            CannotAddAddictionException.class,
```

Comment:
It's a bad practice to add all exceptions into one handler. One exceptionHendler for one exception. Instead of using NoSuchUserExistsException, UserAlreadyExistsException, BadUserCredentialException it's better to use one common exception but with different text message

Reply:
change to one exception for login fail